streams/src/main/java/org/apache/kafka/streams/processor/internals/RecordQueue.java

```diff
@@ -100,6 +104,8 @@ public int addRawRecords(Iterable<ConsumerRecord<byte[], byte[]>> rawRecords, Ti
                                                                          rawRecord.serializedValueSize(), key, value);
             long timestamp = timestampExtractor.extract(record);
 
+            log.trace("source node {} Extracted timestamp {} for record {} when adding to buffered queue", source.name(), timestamp, record);
```

Comment:
Nit:
`log.trace("Source node {}: extracted timestamp {} for record {} when adding to buffered queue", source.name(), timestamp, record);`